code/controllers/voting.dm

```diff
@@ -20,9 +20,13 @@ var/global/list/round_voters = list() //Keeps track of the individuals voting fo
 			qdel(vote)
 		vote = src
 	spawn(0)
-		while(TRUE)
-			process()
-			sleep(10)
+		while(!gcDestroyed)
+			try

```

Comment:
I don't see the point of this.

It ruins stack traces and the stack trace tells you that it's in the voting controller anyways.